config/locales/en.yml

```diff
@@ -53,6 +70,25 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    batch_subscription_notification:
+      subject_linker: "and %{extra} more"
+      chapter_summary: "Chapter Summary:" 
+      text:
+        footer: "You're receiving this email because you've subscribed to %{who}. Follow the link to unsubscribe if you no longer wish to receive updates: %{url}"
+        collected_by: "by %{pseud}"
+        new_chapter_of: "%{byline} posted a new chapter of %{title}:\n%{link}\n\n\
+            %{chapter_title} (%{word_count} words)"
+        work_backdated: "%{byline} backdated a new work:\n%{link}\n\n\
+            %{title} ((%{word_count} words)"
+        work_new: "%{byline} posted a new work:\n%{link}\n\n\
+            %{title} (%{word_count} words)"
+      html:
+        footer: "You're receiving this email because you've subscribed to %{who}. Follow the link to unsubscribe if you no longer wish to receive updates."
+        work_new: "<p>%{byline} posted a new work:<br><b></i>%{link}</i></b></p><p>%{title} (%{word_count} words)</p>"
```

Comment:
Instead of putting the bold and italics around the work title (`%{link}`) here, let's take the same approach we did in delete_work_notification: `<% pretty_title = "<i>#{style_bold(@work.title.html_safe)}</i>".html_safe %>` 

That will cut down on the potential for HTML-related errors in the translation process, e.g. the mistake on the following line, where it currently says `</i>%{link}</i>`.

For reference: https://github.com/otwcode/otwarchive/blob/1fd2e9040e430ee6453530e1ef9ed69a47df35f6/app/views/user_mailer/delete_work_notification.html.erb#L2